src/libs/ajax/WorkspaceDataService.ts

```diff
@@ -0,0 +1,25 @@
+import _ from 'lodash/fp'
+import { authOpts, fetchWDS, jsonBody } from 'src/libs/ajax/ajax-common'
+import { SearchRequest } from 'src/libs/ajax/data-table-providers/WDSDataTableProvider'
+
+
+export const WorkspaceDataService = signal => ({
+  getSchema: async (instanceId: string) => {
+    const res = await fetchWDS(`${instanceId}/types/v0.2`, _.merge(authOpts(), { signal }))
+    return res.json()
+  },
+  getRecords: async (instanceId: string, recordType: string, parameters: SearchRequest) => {
+    const res = await fetchWDS(`${instanceId}/search/v0.2/${recordType}`,
+      _.mergeAll([authOpts(), jsonBody(parameters), { signal, method: 'POST' }]))
+    return res.json()
+  },
+  deleteTable: async (instanceId: string, recordType: string) => {
+    const res = await fetchWDS(`${instanceId}/types/v0.2/${recordType}`,
+      _.mergeAll([authOpts(), { signal, method: 'DELETE' }]))
+    return res
```

Comment:
To make methods more consistent about returning response data vs the Response object, does this need to return the Response?

```suggestion
  deleteTable: async (instanceId: string, recordType: string): Promise<void> => {
    await fetchWDS(`${instanceId}/types/v0.2/${recordType}`,
      _.mergeAll([authOpts(), { signal, method: 'DELETE' }]))
```